charts/aws-vpc-cni/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: aws-vpc-cni
-version: 1.2.2
+version: 1.2.1
```

Comment:
Is this just to revert back the charts version?

Reply:
Yeah, this is a revert to keep in sync with what was shipped with the current `appVersion: "v1.12.0"`. No chart was ever released with version `1.2.2`, so I figured it was best to revert back to the previous state.